nixos/modules/programs/virt-manager.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.virt-manager;
+in {
+  options.programs.virt-manager = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Enables virt-manager, an UI for managing virtual machines in libvirt.
+      '';
+    };
+
+    package = lib.mkOption {
```

Comment:
Please consider `lib.mkPackageOptionMD`

Reply:
> Please consider `lib.mkPackageOptionMD`

No need for `MD` as 23.11 is coming up soon and I doubt this will need backported before then

Reply:
`lib.mkPackageOptionMD` is an alias for `lib.mkPackageOption`, added